R/subOptimalNodeCentricSteinerTreeProblem-class.R

```diff
@@ -123,21 +123,32 @@ subOptimalSteinerProblem <- R6Class("subOptimalSteinerProblem",
         super$solve()
         multiSteinerItr %<>% add(1)
         
+
+
+        if(vcount(super$getCurrentSolutionGraph()) == 0) { 
+        
+        message("STOP iteration, solution not found. No more novelty constraint added")
+        
```

Comment:
Stop iteration when the solver reach the time limit. There's no point to run the same thing over and over again.